bridge/client/app/_components/ktb-heatmap/ktb-heatmap-tooltip.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="tooltip-container" *ngIf="tooltip">
+  <dt-key-value-list>
+    <dt-key-value-list-item>
+      <dt-key-value-list-key> Value </dt-key-value-list-key>
+      <dt-key-value-list-value>
+        {{ tooltip.value | truncateNumber: 2 }}
```

Comment:
If the `tooltip.type` is `IHeatmapTooltipType.SLI` shouldn't we use here `formatNumber(value)` as well instead of the `truncateNumber` pipe?

Reply:
Done. For the `score` this would not change anything, so it does not matter that `formatNumber` is applied to both